ni/measurementlink/sessionmanagement/v1/session_management_service.proto

```diff
@@ -106,6 +109,10 @@ message SessionInformation{
   // This field is readonly.
   bool session_exists = 5;
 
+  // Indicates whether the session is reserved in the Session Manager.
```

Comment:
Maybe update the documentation for the `session_exists` field to say:

Indicates whether the session has been created and registered with the Session Manager.